docs/admin_api/event_reports.rst

```diff
@@ -0,0 +1,57 @@
+Shows reported events
+====================
+
+This API returns information about reported events.
+
+The api is::
+
+    GET /_synapse/admin/v1/event_reports?from=0&limit=10
+
+To use it, you will need to authenticate by providing an ``access_token`` for a
+server admin: see `README.rst <README.rst>`_.
+
+It returns a JSON body like the following:
+
+.. code:: json
+
+    {
+        "event_reports": [
+            {
+                "content": "{\"reason\": \"foo\", \"score\": -100}",
+                "event_id": "$bNUFCwGzWca1meCGkjp-zwslF-GfVcXukvRLI1_FaVY",
+                "id": 2,
+                "reason": "foo",
+                "received_ts": 1570897107409,
+                "room_id": "!ERAgBpSOcCCuTJqQPk:matrix.org",
+                "user_id": "@foo:matrix.org"
+            },
+            {
+                "content": "{\"score\":-100,\"reason\":\"bar\"}",
+                "event_id": "$3IcdZsDaN_En-S1DF4EMCy3v4gNRKeOJs8W5qTOKj4I",
+                "id": 3,
+                "reason": "bar",
+                "received_ts": 1598889612059,
+                "room_id": "!eGvUQuTCkHGVwNMOjv:matrix.org",
+                "user_id": "@bar:matrix.org"
+            }
+        ],
+        "next_token": "2",
+        "total": 4
+}
+
+To paginate, check for ``next_token`` and if present, call the endpoint again
+with from set to the value of ``next_token``. This will return a new page.
+
+If the endpoint does not return a ``next_token`` then there are no more
+users to paginate through.
+
+**URL parameters:**
+
+- ``limit``: Is optional but is used for pagination,
```

Comment:
Do you think it would be useful to also have a `dir` parameter for direction, like we do with for [/rooms](https://github.com/matrix-org/synapse/blob/develop/docs/admin_api/rooms.md#parameters)?

I could see someone wanting to list reports by latest rather than paginating from the oldest each time.